scripts/ci-aws-cred-test.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+
+GROUP_NAME=$(openssl rand -hex 8)
```

Comment:
a prefix or suffix here would make it easier to identify the potentially orphaned resources for someone manually cleaning up the account.

The alternative would be a trap with some deletes, but that can get annoying to write for sure.

Reply:
Good idea. Just pushed changes for that.